frontend/common/stores/project-store.js

```diff
@@ -162,6 +169,24 @@ const store = Object.assign({}, BaseStore, {
     })
   },
   getEnvs: () => store.model && store.model.environments,
+  getMaxFeaturesAllowed: () => {
+    return store.model && store.model.max_features_allowed
+  },
+  getMaxSegmentOverridesAllowed: () => {
+    return store.model && store.model.max_segment_overrides_allowed
```

Comment:
- Can we update responses.ts to include these new types?
- It looks like this is quite a big breaking change, right now e2e is failing because all of these fields are undefined. 
- Should these be controlled via feature flag or assume undefined means Number.MAX_SAFE_INTEGER?